packages/interactivity/src/directives.js

```diff
@@ -57,16 +53,6 @@ export default () => {
 		{ priority: 5 }
 	);
 
-	// data-wp-body
-	directive( 'body', ( { props: { children }, context: inherited } ) => {
-		const { Provider } = inherited;
-		const inheritedValue = useContext( inherited );
-		return createPortal(
-			<Provider value={ inheritedValue }>{ children }</Provider>,
-			document.body
-		);
-	} );
-
```

Comment:
I see the `body` directive being removed. I guess we've lost those changes somehow while moving the files to their own package. 🤔

Reply:
@artemiomorales, could you please take a look at commit https://github.com/WordPress/gutenberg/pull/50906/commits/bec4852a2057211d97bad5643cce3161d0d44ada and check if everything related to the `data-wp-body` directive is there? 🙏 

There was a problem after rebasing this branch onto `trunk`, and some changes were lost.